tests/foreman/ui/test_activationkey.py

```diff
@@ -1227,3 +1227,39 @@ def test_positive_ak_with_custom_product_on_rhel6(session, rhel6_contenthost, ta
         ak = session.activationkey.read(ak.name, widget_names='content_hosts')
         assert len(ak['content_hosts']['table']) == 1
         assert ak['content_hosts']['table'][0]['Name'] == rhel6_contenthost.hostname
+
+
+@pytest.mark.rhel_ver_list([7, 8, 9])
```

Comment:
there is no content host fixture is in use for this test, so why use rhel_ver_list?

Reply:
i will remove